Exercism/DifferenceOfSquares/DifferenceOfSquares.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class Squares
+{
+    public static double SquareOfSums(int max) => Math.Pow(Enumerable.Range(1, max).Sum(), 2);
```

Comment:
Worth considering whether it needs to be `public` - after all, its never used directly.